packages/atomic/src/components/common/triStateCheckbox.spec.ts

```diff
@@ -0,0 +1,100 @@
+import {renderFunctionFixture} from '@/vitest-utils/testing-helpers/fixture';
+import {page} from '@vitest/browser/context';
+import '@vitest/browser/matchers.d.ts';
+import {html} from 'lit';
+import {vi} from 'vitest';
```

Comment:
```suggestion
import {vi, expect} from 'vitest';
```
to not have errors
<img width="1054" alt="image" src="https://github.com/user-attachments/assets/a49a94ac-1e23-4101-9fa6-da4d1c6f5a96" />